Makefile

```diff
@@ -98,6 +103,41 @@ zip: compile transcompile
 	find /tmp/$(PLUGINNAME) -iname "*.pyc" -delete
 	cd /tmp; zip -9r $(CURDIR)/$(PLUGINNAME).zip $(PLUGINNAME)
 
+installer:
+	#Prerequisite is to have a plugin zip ready in the ThreeDiToolbox repo folder.
+	@echo
+	@echo "---------------------------"
+	@echo "Creating Windows Installer."
+	@echo "---------------------------"
+	mkdir ./$(INSTALLER_BUILDDIR)
+	git clone --branch master --depth 1 \
+		git@github.com:nens/3Di-modeller-interface-installer.git \
+		./$(INSTALLER_BUILDDIR)
+	unzip $(PLUGINNAME).zip -d ./$(INSTALLER_BUILDDIR)/$(INSTALLER_PLUGINDIR)/
+	cd ./$(INSTALLER_BUILDDIR); docker build -t 3dimi-installer:latest .
+	git clone --branch master --depth 1 \
+		git@github.com:nens/ThreeDiCustomizations.git \
+		./$(INSTALLER_BUILDDIR)/$(INSTALLER_PLUGINDIR)/ThreeDiCustomizations
+	git clone --branch $(QGIS_VERSION) --depth 1 \
+		git@github.com:qgis/qgis.git ./$(INSTALLER_BUILDDIR)/QGIS
+	docker run \
+	    -v $(shell pwd)/$(INSTALLER_BUILDDIR)/QGIS:/installer/QGIS \
+	    -v $(shell pwd)/$(INSTALLER_BUILDDIR)/3Di-additions:/installer/3Di-additions \
+		-it -e PYTHONUNBUFFERED=0 3dimi-installer ./create_qgis_3di_nsis.pl
+	#docker run \
+	#	-v //d/dev/git/ThreeDiToolbox/$(INSTALLER_BUILDDIR)/QGIS:/installer/QGIS \
+	#	-v //d/dev/git/ThreeDiToolbox/$(INSTALLER_BUILDDIR)/3Di-additions:/installer/3Di-additions \
+	#	-it -e PYTHONUNBUFFERED=0 3dimi-installer ./create_qgis_3di_nsis.pl
+	cp ./$(INSTALLER_BUILDDIR)/QGIS/ms-windows/*3Di*.exe $(CURDIR)/
+
+clean-installer:
+	@echo
+	@echo "-------------------------------------------"
+	@echo "Removing Installer build files and folders."
+	@echo "-------------------------------------------"
+	rm -r $(INSTALLER_BUILDDIR)
+	rm *.exe
```

Comment:
Clean up is now not very useful as it will prompt the user many times and several files cannot be deleted as they are root files from the docker-file.

![image](https://user-images.githubusercontent.com/11858775/69611692-a1f5c500-102e-11ea-8bec-06a6c8aec736.png)